tests/dialects/builtin_test.py

```diff
@@ -0,0 +1,30 @@
+from xdsl.dialects.builtin import *
```

Comment:
I know this is not yet the case in all of the repo, but *-imports are generally considered bad style as there can be shadowing and other issues. Please import only what you need.